lib/ezcater_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EzcaterRubocop
-  VERSION = "0.58.3"
+  VERSION = "0.58.4"
```

Comment:
I lean away from including version bumps directly in PRs, instead preferring to commit the version bump change directly to the releasing branch after it has been merged. This allows for Git history to be consistent when 1 change is released on its own and when multiple changes are released together, as well as limiting the PR content to code that is purely related to the change.

Reply:
This flip side of this is that I like to see what kind of version bump the author intends to make, but hopefully that can be reflected in the CHANGELOG (as it is here).